<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Point_inside_polyhedron</class>
 <widget class="QDockWidget" name="Point_inside_polyhedron">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>225</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Point Inside Facegraph</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="Sample_random_points_from_bbox">
      <property name="text">
       <string>Sample Random Points from Bbox</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Point Location</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QCheckBox" name="Inside_check_box">
           <property name="text">
            <string>Inside</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="On_boundary_check_box">
           <property name="text">
            <string>On Boundary</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="Outside_check_box">
           <property name="text">
            <string>Outside</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="Select_button">
         <property name="text">
          <string>Select</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Note:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;A selection is possible only if a point set item&lt;br/&gt;and a (closed) surface mesh item are selected&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
